js/packages/i18n/locale/en-US/messages.json

```diff
@@ -126,10 +126,6 @@
 			"search": {
 				"no-results": "No results found"
 			},
-			"multi-account": {
-				"create-button": "Create new account",
-				"import-button": "Import account"
-			},
```

Comment:
Maybe we should avoid moving keys around as it will likely break translations in other languages

Reply:
these specific keys are no longer used, but i'll avoid moving used keys 👍